packages/sdk/src/types/Loyalties.ts

```diff
@@ -0,0 +1,500 @@
+	category?: string
+	type: 'LOYALTY_CARD'
+	loyalty_card: {
+		points: number
+		balance: number
+	}
+	start_date?: string
+	expiration_date?: string
+	validity_timeframe?: {
+		interval?: string
+		duration?: string
+	}
+	validity_day_of_week?: number[]
+	publish?: {
+		count?: number
+		entries?: string[]
+	}
+	redemption?: {
+		quantity?: number
+		redeemed_points?: number
+		redeemed_quantity?: number
+		redemption_entries?: string[]
+	}
+	active?: boolean
+	additional_info?: string
+	metadata?: Record<string, any>
+	is_referral_code?: boolean
+	holder_id?: string
+	updated_at?: string
+}
+
+export interface LoyaltiesListMembersResponse {
+	object: 'list'
+	total: number
+	data_ref: 'vouchers'
+	vouchers: LoyaltiesVoucherResponse[]
+}
+
+export interface LoyaltiesCreateMember {
+	voucher?: string
+	channel?: string
+	customer: {
+		id?: string
+		name?: string
+		email?: string
+		metadata?: Record<string, any>
+		description?: string
+		source_id?: string
+	}
+	metadata?: Record<string, any>
+}
+
+export type LoyaltiesCreateMemberResponse = LoyaltiesVoucherResponse
+
+export type LoyaltiesGetMemberResponse = LoyaltiesCreateMemberResponse
+
+export interface LoyaltiesGetMemberActivitiesResponse {
+	object: 'list'
+	data_ref: 'activities'
+	total: number
+	activities: {
+		id: string
+		type: string
+		object: string
+		created_at: string
+		data: any
+	}[]
+}
+
+export interface LoyaltiesAddPoints {
+	points: number
+}
+
+export interface LoyaltiesAddPointsResponse {
+	points: number
+	total: number
+	balance: number
+	type: string
+	object: 'balance'
+	related_object?: {
+		type?: string
+		id?: string
+	}
+}
+
+export interface LoyaltiesRedeemReward {
+	reward: {
+		id: string
+	}
+	metadata?: Record<string, any>
+}
+
+export interface LoyaltiesRedeemOrder {
+	id?: string
+	source_id?: string
+	amount: number
+	items?: OrdersItem[]
+	metadata?: Record<string, any>
+}
+
+interface CoinReward {
+	assignment_id: string
+	loyalty_tier_id: string
+	id: string
+	name: string
+	created_at: string
+	updated_at?: string
+	parameters: {
+		automation_id?: string
+		coin: {
+			exchange_ratio: number
+			points_ratio: number
+		}
+	}
+	type: 'COIN'
+	object: 'reward'
+}
+
+interface MaterialReward {
+	assignment_id: string
+	loyalty_tier_id: string
+	product: ProductsCreateResponse & ProductsCreateSkuResponse
+	id: string
+	name: string
+	created_at: string
+	updated_at?: string
+	parameters: {
+		automation_id?: string
+		product?: {
+			id: string
+		}
+	}
+	type: 'MATERIAL'
+	object: 'reward'
+}
+
+interface CampaignReward {
+	assignment_id: string
+	loyalty_tier_id: string
+	voucher: VouchersResponse
+	id: string
+	name: string
+	created_at: string
+	updated_at?: string
+	parameters: {
+		automation_id?: string
+		campaign?: {
+			id: string
+		}
+	}
+	type: 'CAMPAIGN'
+	object: 'reward'
+}
+
+export interface LoyaltiesRedeemRewardResponse {
+	id: string
+	object: 'redemption'
+	date: string
+	customer_id: string
+	amount: number
+	order: {
```

Comment:
How did you come to the conclusion that `order` is always returned?
I can do request like so:
```
voucherify.loyalties
	.redeemReward("camp_dVpfvCZHgG5iIRdaxXfMtOhl", "v_g4F7aQAjng4kpb6yKjdwjDgM8m36v1im", {
		reward: {
			id: "rew_w63Ju0USU49FcGZ4ZhtxUq6b"
		}
	})
	.then(voucher => {
		console.log('OKS: %j', voucher)
	})
	.catch(error => {
		console.log(error)
	})
```

and it does not return `order` in response. Can't we reuse type from Orders?